packages/perseus/src/util.test.ts

```diff
@@ -5,3 +5,18 @@ describe("firstNumericalParse", () => {
         expect(Util.firstNumericalParse("6/8")).toBe(0.75);
     });
 });
+
+describe("stringArrayOfSize", () => {
+    it("makes an array of strings", () => {
+        expect(Util.stringArrayOfSize(2)).toEqual(["", ""]);
+    });
+});
+
+describe("stringArrayOfSize2D", () => {
+    it("makes a 2D array of strings", () => {
+        expect(Util.stringArrayOfSize2D(2, 4)).toEqual([
```

Comment:
It seems slightly weird to have the number of rows be the first argument, since coordinates are normally `(x, y)`. I might prefer to use an object to name the arguments: `Util.stringArrayOfSize({rows: 2, cols: 4})`.

Alternatively, an API like this might be more transparent and reusable:

```ts
times(2, () => times(4, () => ""))
```